src/transformers/models/bert_japanese/tokenization_bert_japanese.py

```diff
@@ -133,6 +137,7 @@ class BertJapaneseTokenizer(PreTrainedTokenizer):
     def __init__(
         self,
         vocab_file,
+        spm_file,
```

Comment:
```suggestion
        spm_file=None,
```
Needs to be optional or it would be breaking.